site/cds_rdm/templates/semantic-ui/cds_rdm/records/detail.html

```diff
@@ -0,0 +1,32 @@
+{#
+    Copyright (C) 2025 CERN.
+  
+    CDS-RDM is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- extends "invenio_app_rdm/records/detail.html" %}
```

Comment:
this is the reference to my first comment about using include instead of extend, but we could discuss in person about how to shape it

Reply:
I don't think we can do that as with the include we cannot override blocks inside the temaplate, form the [docs](https://jinja.palletsprojects.com/en/stable/templates/#include):

> The included template can extend another template and override blocks in that template. However, the current template cannot override any blocks that the included template outputs.

Reply:
DIscussed IRL, this we keep